bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -118,6 +123,15 @@
   public static final boolean ENABLE_GRPC_RETRY_DEADLINEEXCEEDED_DEFAULT = true;
 
   /**
+   * Key to set to a boolean flag indicating the maximum amount of time to wait for retries, given
+   * a backoff policy on errors.
+   * This flag is used only when grpc retries is enabled.
+   */
+  public static final String MAX_ELAPSED_BACKOFF_MILLIS_KEY =
+      "google.bigtable.grpc.retry.max.elapsed.backoff.millis";
+  public static final int MAX_ELAPSED_BACKOFF_MILLIS_DEFAULT = 180000; // 3 minutes
```

Comment:
below "30 * 60 * 1000" is used  for BIGTABLE_CHANNEL_TIMEOUT_MS_DEFAULT. Could you change it to match  MS/MILLIS and how the number is expressed?